### ISSUE TITLE
《易于理解》

### ISSUE THREAD

**Issue by 394510288** [yatengLG/pointnet2#1]
点云可视化部分打个比方就是：我有碟醋谁有饺子

**Comment by ittong** [comment]
> 点云可视化部分打个比方就是：我有碟醋谁有饺子
您好，我是小白，请问这个代码可视化呀？

**Comment by 394510288** [comment]
> 

import matplotlib.pyplot as plt
from mpl_toolkits.mplot3d import Axes3D
import torch


def plot_3d(xyz, classes=None, s=0.5):
    """
    点云数据可视化
    :param xyz:     xyz坐标
    :param classes: 对应类别
    :param s:       绘图散点大小
    :return:
    """
    assert isinstance(xyz, torch.Tensor) and (xyz.dim()==2)
    if classes is not None:
        assert isinstance(classes, torch.Tensor) and (classes.dim()==1)
        fig = plt.figure()
        ax = fig.gca(projection='3d')
        ax.scatter(xyz[:, 0], xyz[:, 1], xyz[:, 2], s=s, c=classes)
        plt.show()
        return True
    else:
        fig = plt.figure()
        ax = fig.gca(projection='3d')
        ax.scatter(xyz[:, 0], xyz[:, 1], xyz[:, 2], s=s)
        plt.show()
        return True

a = torch.randn(2048,3)
plot_3d(a)


示例

**Comment by ittong** [comment]
> > 
> 
> import matplotlib.pyplot as plt from mpl_toolkits.mplot3d import Axes3D import torch
> 
> def plot_3d(xyz, classes=None, s=0.5): """ 点云数据可视化 :param xyz: xyz坐标 :param classes: 对应类别 :param s: 绘图散点大小 :return: """ assert isinstance(xyz, torch.Tensor) and (xyz.dim()==2) if classes is not None: assert isinstance(classes, torch.Tensor) and (classes.dim()==1) fig = plt.figure() ax = fig.gca(projection='3d') ax.scatter(xyz[:, 0], xyz[:, 1], xyz[:, 2], s=s, c=classes) plt.show() return True else: fig = plt.figure() ax = fig.gca(projection='3d') ax.scatter(xyz[:, 0], xyz[:, 1], xyz[:, 2], s=s) plt.show() return True
> 
> a = torch.randn(2048,3) plot_3d(a)
> 
> 示例

学长你好，能否加个QQ呀，有些网络的原理想请教一下学长，QQ设有问题。
